config/application.rb

```diff
@@ -14,17 +14,14 @@ class Application < Rails::Application
     config.autoload_paths += %W(#{config.root}/app)
```

Comment:
Added this back @carpodaster. In Rails 5, the whole `app` directory is already autoloaded 🤷‍♂️. Forgot that this app uses `spring` which afaik somehow interferes with it.

I'll sort this out in a separate PR. With using `bootsnap` (which works like charm) one could probably even ditch `spring` without any performance / speed impact. I'll see.